.github/workflows/changelog.yml

```diff
@@ -14,19 +14,20 @@ jobs:
 
       - name: Get Changed Files
         id: files
-        uses: Ana06/get-changed-files@v2.1.0
+        uses: tj-actions/changed-files@v34
```

Comment:
Ah this might be the issue. It seems like the quoting in this isn't actually correct.

Reply:
Yeah, `tj-actions/changed-files` seems to escape  `"` by default when we use JSON output. And I didn't find an option to disable it. I could use `sed` to replace `\"` by `"`, but I IMO that's too cumbersome.

Anyway, I've experimented not using JSON output, and instead I'm using the `;` character as the file separator and it worked. I'll push the changes soon.